src/bin/vip-wp.js

```diff
@@ -72,6 +112,64 @@ commandWrapper( {
 
 		let result;
 		let rl;
+		let subShellRl;
+
+		if ( isSubShell ) {
+			console.log( `Entering WP-CLI subshell mode for ${ formatEnvironment( envName ) } on ${ appName } (${ appId })` );
+
+			subShellRl = readline.createInterface( {
+				input: process.stdin,
+				output: process.stdout,
+				terminal: true,
+				prompt: `${ appName }.${ getEnvIdentifier( opts.env ) }> `,
```

Comment:
To make this more "shell-like", perhaps we should use `$` in the prompt instead of `>`? That would make it feel like a native SSH session.

```suggestion
				prompt: `${ appName }.${ getEnvIdentifier( opts.env ) }$ `,
```

Reply:
yeah, this is better 👍